src/main/io/serial_msp.c

```diff
@@ -713,9 +714,51 @@ void mspInit(serialConfig_t *serialConfig)
 
 #define IS_ENABLED(mask) (mask == 0 ? 0 : 1)
 
+static uint32_t packFlightModeFlags(void)
```

Comment:
Yes, it's new MSP_STATUS_EX message to report extended system status,
including system load.